packages/block-library/src/separator/index.js

```diff
@@ -9,7 +9,7 @@ export const name = 'core/separator';
 export const settings = {
 	title: __( 'Separator' ),
 
-	description: __( 'Insert a horizontal line where you want to create a break between ideas.' ),
+	description: __( 'Create a break between ideas or sections with a horizontal line.' ),
```

Comment:
It's not always a line. Even a core style uses 3 dots.

Reply:
Good point. How about:

> Create a break between ideas or sections with a horizontal separator.